### PR TITLE
Expedition Bestiary: guidebook entries and mob tweaks

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
- First draft of extended description for expedition game mode.
- Very rough draft layout of guidebook entries for expedition enemies (only aberrant flesh is somewhat filled out, the templae most likely will be changed, the rest left as blanks).
- Some bundled changes to monsters to make them slightly more diverse and comply with "emerging narrative" (prepping guidebook entries gave some insight).

## To do:
- [x] Copy mob descriptions from the guidebook to entities after **_somebody_** (@whatston3) proofreads them.

## Bundled general changes:
- Counterfeit spesos: spessos. For now just a place holder that also stacks with real spesos. Made functional by @whatston3. Original sprites by Stagnation (discord), tweaked slightly by @whatston3.
- Added some new flatpacks.
- Added 50 minutes despawn timer to arcadia industries mobs (max blue space event duration involving those mobs - 40 mins).
- Added 30 minutes despawn timer to wizard federation mobs (max blue space event duration involving these mobs - 20 mins).

### General changes:
- Corrected `speedModifierThresholds` for `SlowOnDamage` for expedition mobs.
- Removed all access from expedition mobs.
- Buffed loot from NPCs.
- Added flatpacks to loot pools.
- Added restock boxes to loot pools.

### Aberrant flesh:
- Changed parent's damage type to blunt.
- Gave `MobFleshLoverExpeditions` the ability to fly.
- Added new mob: partially assimilated miner. Unremarkable, has 5% spawn chance.
- New destruction objective: assimilation sack. Sprites by **Stagnation** (discord). Has an 85% chance to spawn 1 to 4 aberrant flesh mobs every 4 minutes.
- Fixed solution transfer on bone spikes. _**Again.**_

### Argocytes:
- Tweaked damage thresholds and movement speeds in order to clearly define 2 morphotypes within the species: slow tanky "guardians", nimble and fragile "stalkers".
- "Guardians" deal blunt damage, "stalkers" deal slash damage.
- Can be butchered for meat and construction materials (in token quantities).

### Blood cult:
- Ascended cultist: has proper flier fixtures, improved visuals a bit.
- Drained one: has increased lifespan, now incorporeal (like holocarp), but deals less damage. Drained one summoned by players has increased lifespan and will follow its summoner.
- Added shade: same as drained one, but deals slash and blood loss damage types, has a 5% chance to be summoned by ascended cultist instead of the drained one. Sprites taken from https://github.com/NebulaSS13/Nebula/blob/dev/icons/mob/simple_animal/shade.dmi (CC-BY-SA-3.0 afaict).
- Added 50 minutes despawn timer (max blue space event duration involving those mobs - 40 mins).

### Carps:
- Improved visuals: added animated unshaded mouth to expedition versions of holocarp and magicarp.

### Dinosaurs:
- Tweaked some damage and movement speed values.
- Changed scaling of compsognathus sprite to be more in line with wiki article (apparently it was slightly shorter than a human in height).

### Explorers:
- Tweaked some damage values. Nerfed fire rates.
- Added damage resistance profiles to T2 and T3 mobs.

### Mercenaries:
- Tweaked some damage values. Nerfed fire rates.
- Added new mob: ranger, armed with gestio rifle.
- New destruction objective: counterfeit cache. Sprites by @gentlebutter (discord/github).

### Punks
- Renamed Gangers -> Punks
- Punk leader has 'Jittering' trait.
- Punks shit graffities.
- New destruction objective: party supplies. Sprites by @gentlebutter (discord/github). A tank, filled with party chemicals.

### Rogue silicons:
- Tweaked some damage values.
- Removed regeneration from drones.
- Added new mob: viscerator unit - small fast flying machine spawned by catcher units. Low hp, low damage, no regeneration. Sprites taken from NebulaSS13 (general purpose SS13/Baystation12 fork) at https://github.com/NebulaSS13/Nebula/blob/dev/icons/mob/simple_animal/viscerator.dmi (CC-BY-SA-3.0 afaict), tweaked slightly (split into layers, made lights less prominent).
- Buffed boss HP.
- Rogue AI node is now electrified.
- Added 20 min despawn timer (because for some reason these mobs are not deleted on planed deletion).

### Syndicate naval forces:
- Fixed solution transfer on darts. _**Again.**_
- Added voidsmen: syndicate naval troopers equipped in EVA/hardsuits for use in bluespace events.
- Added 50 minutes despawn timer (max blue space event duration involving those mobs - 40 mins).

## Why / Balance
To give players information about game mechanics tied to expedition monsters.

## How to test
Open the guidebook

## Media
image
image
image
image
image
image
image
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
Shouldn't be any

**Changelog**
:cl: erhardsteinhauer
- add: Added guidebook entries for expedition enemies. Proofread by @whatston3.
- add: Added unique destruction target for Aberrant Flesh expedition faction - assimilation sack (sprites by Stagnation).
- add: Added unique destruction targets for Punks (party supplies) and Mercenaries (counterfeit cache) expedition factions. Sprites by GentleButter (BING GUS!).
- add: Added counterfeit spesos - spessos. Functionality by @whatston3. Original sprites by Stagnation (discord), tweaked slightly by @whatston3.
- tweak: Expanded expeditions loot pool with flatpacks and vending machines restock boxes.
- tweak: Buffed loot from hostile NPCs: more cash, mats, parts, sometimes flatpacks and weapon cases.
- tweak: Renamed Gangers into Punks.
- tweak: Tweaked (a little bit) damage values, damage types, movement speeds and health pools for expedition mobs.
- tweak: Removed health regeneration from rogue AI drones.
- fix: Expedition mobs will now slow down upon taking appropriate amount of damage (on 60% and 80% of mob's total health).

### CODE REVIEW

**Comment by ErhardSteinhauer** [comment]
The PR is ready, all checks have passed. Going to put it back in draft until https://github.com/new-frontiers-14/frontier-station-14/pull/1534 is merged

File: Resources/ServerInfo/_NF/Guidebook/Expeditions.xml [hunk 1]
```diff
@@ -1,33 +1,170 @@
...
+  <GuideEntityEmbed Entity="ClothingHeadsetGrey" Caption="headset"/>
+  <GuideEntityEmbed Entity="EncryptionKeyMedical"/>
+  <GuideEntityEmbed Entity="MedicalTrackingImplanter"/>
+  </Box>
+  
+  Communication is crucial. Make sure to keep your teammates informed about your status and inquire about their well-being. To maintain contact with your crew, you need either a functioning [color=#a4885c]telecommunication server[/color] with [color=#a4885c]encryption keys[/color] installed, or distribute [color=#a4885c]handheld radios[/color] among crew members (and ensure they are turned on).
+
+  Additionally, having [color=#a4885c]medical encryption keys[/color] installed in your and your crew's headsets, as well as [color=#a4885c]medical tracking implanters[/color] applied, will enable instant notification if one of you goes down in combat.
+  
+  ## Know your enemy
```
Comment (whatston3) [inline Resources/ServerInfo/_NF/Guidebook/Expeditions.xml:163]:
```suggestion
  Communication is crucial. Make sure to keep in contact with your teammates to know what they're doing and when they need help. You'll need either a headset and a [color=#a4885c]telecommunication server[/color] with [color=#a4885c]encryption keys[/color] installed, or [color=#a4885c]handheld radios[/color] on each crewmate (make sure they're on!).

  Having a [color=#a4885c]medical encryption key[/color] installed in your headset will tell you when anyone with a  [color=#a4885c]medical tracking implanter[/color] installed goes down in combat.  Leave no friend behind!
  
  ## Know Your Enemy
```

File: Resources/ServerInfo/_NF/Guidebook/Expeditions.xml [hunk 2]
```diff
@@ -1,33 +1,170 @@
...
+  <Box>
+  <GuideEntityEmbed Entity="ClothingEyesHudMedical"/>
+  <GuideEntityEmbed Entity="ClothingEyesHudDiagnostic"/>
+  </Box>
+  Adaptability is key: strategies and tactics that prove effective against [color=#a4885c]Xeno[/color] threats may not work at all against hostile [color=#a4885c]mercenaries[/color], and vice versa. Guidebook entries on hostile entities may provide some insight into what to expect planet-side, but they cannot replace firsthand experience. Instead, descend to the planet, salvage what you can, and face the dangers head-on, for the riches won't wait for you forever!
```
Comment (whatston3) [inline Resources/ServerInfo/_NF/Guidebook/Expeditions.xml:168]:
```suggestion
  Adaptability is key: strategies and tactics that work against [color=#a4885c]Xenos[/color] might not work well against [color=#a4885c]mercenaries[/color], and vice versa. The Guidebook entries on hostile entities may provide some insight into what to expect planetside, but they [italic]cannot[/italic] replace firsthand experience. Instead, descend to the planet, salvage what you can, and face danger head-on. The riches won't wait around forever!
```

**Review by whatston3 — CHANGES_REQUESTED** [review]
Partial review.  Still need to look through the enemy type entries, but looks good in-game from what I've seen.  Try to be a bit skeptical on the suggested changes.

Sorry it took so long to get to this.

**Comment by ErhardSteinhauer** [comment]
I probably should use links whenever I mention specific factions in exped overview entry

File: Resources/Prototypes/_NF/Entities/Mobs/NPCs/mob_hostile_base.yml [hunk 3]
```diff
@@ -42,7 +42,7 @@
     - Alive
     - Critical
     - Dead
-    damageCap: 400
+    damageCap: 9999
```
[the remark this answers is not in the document]
Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Mobs/NPCs/mob_hostile_base.yml:45]: No, it's there to make sure that health regen will be "always" active, no matter how far in the negatives mobs health is.

File: Resources/Prototypes/_NF/Entities/Mobs/NPCs/mob_hostile_expeditions_aberrant_flesh.yml [hunk 4]
```diff
@@ -147,13 +147,70 @@
       Dead:
         Base: dead
 
+- type: entity
+  parent: BaseMobFleshExpeditions
+  id: MobFleshAssimilatedMiner
```
[the remark this answers is not in the document]
Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Mobs/NPCs/mob_hostile_expeditions_aberrant_flesh.yml:152]: Haven't decided yet tbh. The rest of the mobs in this faction don't have unique names, so I kinda left this one without name too

File: Resources/Prototypes/_NF/Entities/Objects/Misc/space_cash.yml [hunk 5]
```diff
@@ -19,3 +19,63 @@
     state: cash_25000
   - type: Stack
     count: 25000
+
+# Fake fake money (ikr), placeholder for now until actual counterfeit currency is implemented
+- type: entity
```
[the remark this answers is not in the document]
Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Objects/Misc/space_cash.yml:24]: Sounds good!

File: Resources/Prototypes/_NF/Entities/Mobs/NPCs/misc.yml [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+  components:
+  - type: Loadout
+    prototypes:
+    - GuidebookMemeNpcUglyLoadout
+
+- type: entity
+  parent:
+  - MobGoblin
+  id: GuidebookMemeNpcThatGuy
+  name: "and that guy"
```
[the remark this answers is not in the document]
Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Mobs/NPCs/misc.yml:40]: Yeah, good call. When I made those I wasn't sure how captions work in xml.

File: Resources/ServerInfo/_NF/Guidebook/Expeditions.xml [hunk 7]
```diff
@@ -1,33 +1,170 @@
...
+  <GuideEntityEmbed Entity="GlowstickBlue"/>
+  <GuideEntityEmbed Entity="FlashlightLantern"/>
+  <GuideEntityEmbed Entity="Floodlight"/>
+  </Box>
+  [bold][color=#a4885c]Time of day:[/color][/bold]
+  - [bold]Morning/Day/Evening times:[/bold] the ambient light on the planet is sufficient, no need for light sources.
+  - [bold]Night time:[/bold] pitch black darkness, light sources are required for navigating the planet.
+  
+  [bold][color=#a4885c]Time constraints:[/color][/bold]
+  - [bold]Rush:[/bold] you will have less time to complete the expedition. Set your priorities in advance and try to avoid wearing too much gear with negative speed modifiers.
```
[the remark this answers is not in the document]
Comment (ErhardSteinhauer) [reply Resources/ServerInfo/_NF/Guidebook/Expeditions.xml:87]: fanning out - might not be the best advice.

File: Resources/Prototypes/_NF/Entities/Structures/Specific/punks.yml [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+        - ReagentId: Desoxyephedrine
+          Quantity: 30
+        - ReagentId: THC
+          Quantity: 30
+        - ReagentId: Happiness
+          Quantity: 10
+        - ReagentId: Stimulants
+          Quantity: 5
+        - ReagentId: Whiskey
+          Quantity: 125
```
[the remark this answers is not in the document]
Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Entities/Structures/Specific/punks.yml]: Added free non-taxable vent crud, space drugs, mopwata and acid spit

File: Resources/ServerInfo/_NF/Guidebook/Expeditions.xml (hunk 7 above)
[the remark this answers is not in the document]
Comment (ErhardSteinhauer) [reply Resources/ServerInfo/_NF/Guidebook/Expeditions.xml]: added the rest

**Comment by dvir001** [comment]
Might want to remove the timers
This should be fixed on despawn code

**Comment by ErhardSteinhauer** [comment]
> Might want to remove the timers This should be fixed on despawn code

Sure.

**Comment by ErhardSteinhauer** [comment]
> Might want to remove the timers This should be fixed on despawn code

Done

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**